miden-tx/src/kernel_tests/test_account.rs

```diff
@@ -23,14 +23,14 @@ use miden_objects::{
         },
     },
 };
-use vm_processor::{ContextId, Felt, MemAdviceProvider};
+use vm_processor::{AdviceInputs, ContextId, DefaultHost, Felt, MemAdviceProvider};
 
 use super::{ProcessState, StackInputs, Word, ONE, ZERO};
 use crate::{
     kernel_tests::output_notes_data_procedure,
     testing::{
         mock_executed_tx, mock_inputs,
-        utils::{prepare_transaction, run_tx, run_within_host, run_within_tx_kernel},
```

Comment:
I removed `run_tx` and `run_within_tx_kernel`, since these were doing the same work as `run_tx_with_inputs`. This removes indirection and makes it easier to understand the tests.